rpcs3/Emu/RSX/VK/VKHelpers.cpp

```diff
@@ -1,4 +1,5 @@
 ﻿#include "stdafx.h"
+#include "stdafx.h"
```

Comment:
What's this include for?

Reply:
No idea, looks like visual studio thought it would be nice to have a duplicate. Probably just accidental Ctrl+D. Will remove.